### DISCUSSION TITLE
Unusual Trend in Prophylactic Anticoagulation Rates in ICU Patients from 2008-2019

### DISCUSSION THREAD

**Discussion by umcgicv** [MIT-LCP/mimic-code#1683] [MIMIC-IV]
Hello,

We are seeking to identify patients who received prophylactic anticoagulation within the first 24 hours of ICU admission. For this analysis, we utilized the following itemids in inputevents: 225152, 229597, 225975, 225906, and 225908, applying specific cutoff values for 225152 and 229597.

During our examination of the data spanning from 2008 to 2016, we observed that approximately 40% of patients received prophylactic anticoagulation within the specified timeframe. However, a striking trend emerged in the subsequent years (2017-2019), where this percentage plummeted to approximately 0.8%.

This abrupt decline raises concerns about potential gaps in our analysis or changes in clinical practices over time. Could there be elements we are overlooking in our data interpretation or external factors influencing this trend?

Any input or guidance would be greatly appreciated.

Thank you in advance for your assistance.

**Comment by nkchangshu** [comment]
How did you ultimately resolve it?

**Reply by umcgicv** [reply]
Hi,

I supplemented inputevents with data from emar. By doing this, the dip seen in years 2017-2019 disappears. Note that emar, in my opinion, should be approached with more caution than inputevents i.e more data cleaning is required.

Hope this was helpful!
